src/main/java/macrobase/analysis/stats/mixture/BatchMixtureModel.java

```diff
@@ -91,7 +100,7 @@ public double getZScoreEquivalent(double zscore) {
     }
 
     public boolean checkTermination(double logLikelihood, double oldLogLikelihood, int iteration) {
-        log.debug("log likelihood after iteration {} is {}", iteration, logLikelihood);
+        log.info("average point log likelihood after iteration {} is {}", iteration, logLikelihood);
```

Comment:
You should be able to change the logging level in your MacroBase yaml conf.
For example, see the server conf.

On Monday, May 16, 2016, Arsen Mamikonyan <notifications@github.com> wrote:

> In src/main/java/macrobase/analysis/stats/mixture/BatchMixtureModel.java
> <https://github.com/stanford-futuredata/macrobase/pull/141#discussion_r63367984>
> :
>
> > @@ -91,7 +100,7 @@ public double getZScoreEquivalent(double zscore) {
> >      }
> >
> >      public boolean checkTermination(double logLikelihood, double oldLogLikelihood, int iteration) {
> > -        log.debug("log likelihood after iteration {} is {}", iteration, logLikelihood);
> > +        log.info("average point log likelihood after iteration {} is {}", iteration, logLikelihood);
>
> debug doesn't show up when I'm running diagnostic tasks. I didn't know how
> to set a not default log level there.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/stanford-futuredata/macrobase/pull/141/files/37d1ddc4af18586ad20ecf82ba162fb3f4d2cc75#r63367984>
>